x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_rule_actions/index.tsx

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiHorizontalRule, EuiFlexGroup, EuiFlexItem, EuiButton, EuiSpacer } from '@elastic/eui';
+import React, { FC, memo, useCallback, useEffect, useMemo, useState } from 'react';
+import deepEqual from 'fast-deep-equal';
+
+import { setFieldValue } from '../../helpers';
+import { RuleStep, RuleStepProps, ActionsStepRule } from '../../types';
+import { StepRuleDescription } from '../description_step';
+import { Form, UseField, useForm } from '../../../../../shared_imports';
+import { StepContentWrapper } from '../step_content_wrapper';
+import { ThrottleSelectField, THROTTLE_OPTIONS } from '../throttle_select_field';
+import { RuleActionsField } from '../rule_actions_field';
+import { useKibana } from '../../../../../lib/kibana';
+import { schema } from './schema';
+import * as I18n from './translations';
+
+interface StepRuleActionsProps extends RuleStepProps {
+  defaultValues?: ActionsStepRule | null;
+  actionMessageParams: string[];
+}
+
+const stepActionsDefaultValue = {
+  enabled: true,
+  isNew: true,
+  actions: [],
+  kibanaSiemAppUrl: '',
+  throttle: THROTTLE_OPTIONS[0].value,
+};
+
+const GhostFormField = () => <></>;
+
+const StepRuleActionsComponent: FC<StepRuleActionsProps> = ({
+  addPadding = false,
+  defaultValues,
+  isReadOnlyView,
+  isLoading,
+  isUpdateView = false,
+  setStepData,
+  setForm,
+  actionMessageParams,
+}) => {
+  const [myStepData, setMyStepData] = useState<ActionsStepRule>(stepActionsDefaultValue);
+  const {
+    services: { application },
+  } = useKibana();
+
+  const { form } = useForm({
+    defaultValue: myStepData,
+    options: { stripEmptyFields: false },
+    schema,
+  });
+
+  const kibanaAbsoluteUrl = useMemo(() => application.getUrlForApp('siem', { absolute: true }), [
+    application,
+  ]);
```

Comment:
Thanks for this fix -- I believe this should be fine for cloud, and shouldn't have any effect with CCS since this functionality is specific to rules/saved objects.